dex/networks/eth/common.go

```diff
@@ -55,7 +55,7 @@ func CheckAPIModules(c *rpc.Client, endpoint string, log dex.Logger, reqModules
 		for v := range reqModulesMap {
 			reqs = append(reqs, v)
 		}
-		return fmt.Errorf("needed apis not present: %v", strings.Join(reqs, " "))
+		return fmt.Errorf("needed apis not present: %v. some APIs may only be available over websocket and not http.", strings.Join(reqs, " "))
```

Comment:
I'm curious if there was a specific case where you found ws had it but on http.  I assume that's solvable with the `--http.api=` flag, unless you were referring to a third-party RPC provider.

I'm afraid we may have to go with the "dumbed down" balance check https://github.com/decred/dcrdex/pull/2020 because I haven't seen any RPC providers (other than geth itself) that provide `txpool`.  e.g. https://docs.alchemy.com/reference/ethereum-api-faq#what-methods-does-alchemy-support-for-the-ethereum-api
Or we could at least fall back to that method when txpool is not available.  Not needed for this PR, but have been thinking.

Reply:
Yeah you're right, I just had to add -http.api.